addons/mail/static/tests/discuss/call/call.test.js

```diff
@@ -247,30 +247,6 @@ test("can share user camera", async () => {
     await contains("video", { count: 0 });
 });
 
-test("Card should remain in focus after Share Screen is toggled", async () => {
-    mockGetMedia();
-    const pyEnv = await startServer();
-    const channelId = pyEnv["discuss.channel"].create({ name: "General" });
-    await start();
-    await openDiscuss(channelId);
-    await click("[title='Start a Call']");
-    await contains(".o-discuss-Call.o-minimized"); // minimized = no card focused
-    await click(".o-discuss-CallParticipantCard-avatar");
-    await contains(
-        ".o-discuss-Call:not(.o-minimized) .o-discuss-CallParticipantCard:not(.o-inset) img" // inset = aside, not inset = center
-    );
-    await click("[title='Share Screen']");
-    await contains(".o-discuss-CallParticipantCard.o-inset img");
-    await contains(
-        ".o-discuss-Call:not(.o-minimized) .o-discuss-CallParticipantCard:not(.o-inset) video[type='screen']"
-    );
-    await click("[title='Stop Sharing Screen']");
-    await contains(".o-discuss-CallParticipantCard.o-inset", { count: 0 });
-    await contains(
-        ".o-discuss-Call:not(.o-minimized) .o-discuss-CallParticipantCard:not(.o-inset) img"
-    );
-});
-
```

Comment:
Test is removed because this is replaced by the other new test, to match the new behaviour.